GVRf/Framework/jni/objects/scene_object_jni.cpp

```diff
@@ -176,7 +179,7 @@ Java_org_gearvrf_NativeSceneObject_attachEyePointeeHolder(
         jlong jeye_pointee_holder) {
     SceneObject* scene_object = reinterpret_cast<SceneObject*>(jscene_object);
     EyePointeeHolder* eye_pointee_holder =
-            reinterpret_cast<EyePointeeHolder*>(jeye_pointee_holder);
+    reinterpret_cast<EyePointeeHolder*>(jeye_pointee_holder);
```

Comment:
I think the format change here seems a little weird to me. Please make sure that you are using K&R built-in with modification on Tab policy(spaces only).